src/modules/web-inbox/helper.js

```diff
@@ -16,13 +16,17 @@ export const processWebInboxSettings = (webInboxSetting, isPreview = false) => {
 }
 
 export const processInboxNotifs = (msg) => {
-  if (msg.inbox_preview) {
```

Comment:
do not remove this code. in case of preview, we need to set messages against incomingMessagesForPreview. If we save them against incomingMessages, they'll end up getting saved in LS, viewed and clicked events will also get raised for these.

Reply:
Fixed this